docs/deployment.md

```diff
@@ -0,0 +1,214 @@
+# Deployment of Forcerelay/Axon
+
+dependencies and tools:
+1. Axon Rpc (Alphanet): https://rpc-alphanet-axon.ckbapp.dev
+2. CKB Rpc (Testnet): https://testnet.ckbapp.dev/
+3. Yarn: v1.22.19
+4. ckb-cli: v1.4.0
+5. cargo: v1.74,0
+
+related repos and commits:
+1. ibc-solidity-contract: https://github.com/synapseweb3/ibc-solidity-contract
+2. forcerelay-ckb-contracts: https://github.com/synapseweb3/forcerelay-ckb-contracts
+3. forcerelay-ckb-sdk: https://github.com/synapseweb3/forcerelay-ckb-sdk
+
+## Deploy `ibc-solidity-contracts`
+deploy from the repo (Yarn needed):
+```bash
+$ git clone https://github.com/synapseweb3/ibc-solidity-contract
+$ cd ibc-solidity-contract
+$ yarn install
+$ yarn compile
+$ AXON_HTTP_RPC_URL=https://rpc-alphanet-axon.ckbapp.dev yarn migrate
+```
+ 
+after the deployment, contracts `OwnableIBCHandler` and `ICS20TransferERC20` are in your output, let's record the addresses of them as **<IBC_HANDLER_ADDRESS>** and **<TRANSFER_ERC20_ADDRESS>** for a later use.
```

Comment:
1. The console.log in the 1_deploy_contracts.js script returns too much print content, making it inconvenient to filter `OwnableIBCHandler` and `ICS20TransferERC20`. It is recommended to simplify; or return the values ​​of the two agreed addresses required for configuration.
```
 return {
    ibcHandlerAddress: ibcHandler.address,
    ics20Transfer address: ics20Transfer.address
  };
```